### PR TITLE
[stable/prometheus-operator] Allow to pass --deny-namespaces to Prometheus Operator

### PR DESCRIPTION
**Pull request by mcanevet**
#### What this PR does / why we need it:
We should be able to pass `--deny-namespaces` to the Prometheus Operator

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by mcanevet** [comment]
@vsliouniaev tests fails but I doubt it comes from my PR

**Comment by zanhsieh** [comment]
@mcanevet 
You need a chart version bump to the latest.

**Comment by mcanevet** [comment]
@zanhsieh done

File: stable/prometheus-operator/values.yaml [hunk 1]
```diff
@@ -964,6 +964,9 @@ prometheusOperator:
       podAnnotations: {}
       nodeSelector: {}
 
+  ## Namespaces not to scope the interaction of the Prometheus Operator (deny list).
```
Comment (zanhsieh) [inline stable/prometheus-operator/values.yaml]: @mcanevet 
Could you figure out why circle-ci chart lint report this error please?

stable/prometheus-operator/values.yaml
  967:5     error    missing starting space in comment  (comments)

Comment (mcanevet) [reply stable/prometheus-operator/values.yaml]: Should be better now.

**Pull request merged**